plugins/excel-adapter/src/main/java/org/polypheny/db/adapter/excel/ExcelTableScan.java

```diff
@@ -78,30 +77,30 @@ public AlgDataType deriveRowType() {
 
 
     @Override
-    public void register( AlgOptPlanner planner ) {
+    public void register( AlgPlanner planner ) {
         //planner.addRule( ExcelProjectTableScanRule.INSTANCE );
```

Comment:
Is there no rule to register?

Reply:
As this only supports Scannable and FilterableEntity, there is no rule to register. So no. Added comment.